### PR TITLE
Fix/bar

### PR DESCRIPTION
**Pull request by GoOz**
Fix all kinds of minor cosmetic flaws with the cozy-bar & top bar:
- [x] Increased bar's height on mobile (see https://github.com/cozy/cozy-bar/pull/40)
- [x] Bottom border was lighter below the more button (see https://github.com/cozy/cozy-ui/pull/96)
- [x] Alignments of every more buttons with the one in the top bar
- [x] Removed margin before app's name on mobile when there's no icon

### CODE REVIEW

**Comment by GoOz** [comment]
[(3) [FIL-38] Cosmétique - Cozy Bar](https://trello.com/c/C8qKQwVL/236-3-fil-38-cosmetique-cozy-bar)

File: src/styles/topbar.styl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @require 'cozy-ui'
 
-$coz-bar-size = 40px
+$coz-bar-size = 48px
```
Comment (kosssi) [inline src/styles/topbar.styl]: 👍

File: src/styles/breadcrumb.styl [hunk 2]
```diff
@@ -32,16 +32,17 @@
...
+    white-space    nowrap
+    line-height    1
 
 @media (max-width: (768/basefont)rem)
     .fil-path-backdrop
         margin   0
         position fixed
         top      0
-        left     $coz-bar-size
+        left     2.5em
```
Comment (m4dz) [inline src/styles/breadcrumb.styl]: Are we sure about the left width? (regarding to the bar, I mean…)

File: src/styles/topbar.styl (hunk 1 above)
Comment (m4dz) [inline src/styles/topbar.styl]: Are we using it so? I see some replace of `$coz-bar-size` by `2.5em`

Comment (GoOz) [inline src/styles/topbar.styl]: Yeah i'm using it elsewhere, I replaced by 2.5em just two of them cause it didn't fit anymore.

File: src/styles/breadcrumb.styl (hunk 2 above)
Comment (GoOz) [inline src/styles/breadcrumb.styl]: What do you mean? I don't follow.

**Comment by m4dz** [comment]
@GoOz it's ok for me. I'va asked myself mainly about replacing `$coz-bar-size` with `2.5em` but you justified it, thanks :+1: !

**Pull request merged**